interfaces/nginx_route/v0/schemas/requirer.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class NginxRouteRequirerSchema(BaseModel):
+    service_hostname: str = Field(
+        ...,
+        alias="service-hostname",
+        description="The hostname of the service to create an ingress for.",
+        examples=["example.com"],
+        title="Service Hostname",
+    )
+    service_name: str = Field(
+        ...,
```

Comment:
is this supposed to mean that the field is mandatory? 
If so, leave the first arg empty. As it is, the default is `typing.Ellipsis`.

Reply:
if you want to do it explicitly, you have to pass `pydantic.fields.Undefined`

Reply:
Yes, `...` is used to indicate that the filed is required in pydantic v1.0.7. I have removed the `...` and using the default instead,

Reply:
oh, my bad, I thought `...` wasn't a valid arg in pydantic.